src/theme-cosmos.css

```diff
@@ -3,28 +3,92 @@ body {
   --ds-color: black;
   --ds-color-alt: white;
   --ds-color-primary: #5064FB;
-  --ds-font-family: "Gotham A", "Gotham B", "Gotham", sans-serif;
+  --ds-font-family: "GothamSSm", "Gotham SSm A", "Gotham SSm B", "Gotham A", "Gotham B", "Gotham", sans-serif;
+
+  --ds-d1-font-size: 5rem;
+  --ds-d1-font-weight: bold;
+  --ds-d1-line-height: 6rem;
+  --ds-d1-letter-spacing: -0.075em;
+
+  --ds-d2-font-size: 4rem;
+  --ds-d2-font-weight: bold;
+  --ds-d2-line-height: 5rem;
+  --ds-d2-letter-spacing: -0.075em;
+
+  --ds-d3-font-size: 3rem;
+  --ds-d3-font-weight: bold;
+  --ds-d3-line-height: 4rem;
+  --ds-d3-letter-spacing: -0.07em;
 
   --ds-h1-font-size: 3rem;
+  --ds-h1-font-weight: bold;
+  --ds-h1-line-height: 3.25rem;
+  --ds-h1-letter-spacing: -0.065em;
 
-  --ds-h2-font-size: 2rem;
+  --ds-h2-font-size: 2.5rem;
+  --ds-h2-font-weight: bold;
+  --ds-h2-line-height: 3.25rem;
+  --ds-h2-letter-spacing: -0.065em;
 
-  --ds-h4-font-size: 1.125rem;
-  --ds-h4-font-weight: 600;
+  --ds-h3-font-size: 1.5rem;
+  --ds-h3-font-weight: bold;
+  --ds-h3-line-height: 2rem;
+  --ds-h3-letter-spacing: -0.05em;
 
-  --ds-h5-font-size: .875rem;
+  --ds-h4-font-size: 1.25rem;
+  --ds-h4-font-weight: bold;
+  --ds-h4-line-height: 2rem;
+  --ds-h4-letter-spacing: -0.045em;
+
+  /* to be removed */
+  --ds-h5-font-size: 0.875rem;
   --ds-h5-font-weight: rgba(0,0,0,.5);
```

Comment:
😕

Reply:
h5 and h6 are currently not on DS Figma. Will update them once the font scale is finalized.